pkg/types/nutanix/platform.go

```diff
@@ -1,20 +1,27 @@
 package nutanix
 
 import (
+	"fmt"
+
 	configv1 "github.com/openshift/api/config/v1"
 )
 
+// CredentialsSecretName is the default nutanix credentials secret name.
```

Comment:
maybe add the nolint comment here?
//nolint:gosec
```
pkg/types/nutanix/platform.go:10:7: G101: Potential hardcoded credentials (gosec)
const CredentialsSecretName = "nutanix-credentials"
```

Reply:
yeah, added.